app/assets/stylesheets/views/_taxons.scss

```diff
@@ -81,3 +81,15 @@
     }
   }
 }
+
+.taxon-page__organisations {
+  padding: $gutter-half;
+}
+
+.taxon-page__organisation {
+  display: inline-block;
+  width: 200px;
+  vertical-align: top;
+  margin-bottom: 10px;
```

Comment:
Have you considered more spacing between rows? It looks a bit squashed at the moment.